tests/v2/validation/Jenkinsfile.e2e

```diff
@@ -111,18 +112,18 @@ node {
               }
               dir ("./") {
                 sh "./rancher/tests/v2/validation/configure.sh"
-                sh "docker build . -f ./rancher/tests/v2/validation/Dockerfile.e2e -t ${golangImageName}"
-                sh "docker volume create --name ${registriesVolume}"
+                sh "docker build . -f ./rancher/tests/v2/validation/Dockerfile.e2e --build-arg CORRAL_VERSION=${env.CORRAL_VERSION} -t ${golangImageName}"
```

Comment:
Just to confirm, `CORRAL_VERSION` will come from a job variable?

Reply:
Yes, that is correct and I'm doing this simply because there hasn't been new corral version yet that changes I need so I got the most recent version based on the latest commit.